sympy/core/sympify.py

```diff
@@ -273,6 +272,31 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
     >>> kernS(s)
     -2*(-(-x + 1/x)/(x*(x - 1/x)**2) - 1/(x*(x - 1/x))) - 1
 
+    Parameters
+    ==========
+
+    a : Can be :
```

Comment:
I'd denote the typing like `a : object` or `a` if the type can be everything.

Reply:
Sure, I will make the changes. The Parameters section has been rendered before the explanation section here, can you spot any mistakes on my part?